proto/store.proto

```diff
@@ -7,6 +7,25 @@ option csharp_namespace = "Momento.Protos.Store";
 
 package store;
 
+// Possible headers returned in grpc error responses
+// {
+//   Key: "err"
+//   Values: [
+//    "momento_general_err",
```

Comment:
Could you add back in the clarifying comments for each of the alternatives we had previously? (as in https://github.com/momentohq/client-protos/pull/292/commits/6a6c3cf5e73710ca1f86b9f1668d7a3f0b582ff2).